prisma/schema.prisma

```diff
@@ -10,24 +10,98 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Exhibitor {
-  id                 String  @id @default(uuid())
+model ExhibitorInterestRegistration {
+  id                 String   @id @default(uuid())
+  createdAt          DateTime @default(now())
   name               String
-  organizationNumber String?
+  organizationNumber String
```

Comment:
How do we resolve the company without an organisation number?

Reply:
I think we can make it 000000-0000, this way there are less edge cases to think about, since we don't do anything more than display the organization number.